### PR TITLE
Task 50 with some refactoring of task 20

### PR DESCRIPTION
**Pull request by MarinaGotka**

### CODE REVIEW

File: QAAutomationTraining/TUT.by/BasePage.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        public void WaitUntilDisplayed(IWebDriver driver, By locator)
+        {
+            //Polling frequency 500 ms is added 
+            var element = new WebDriverWait(clock, driver, TimeSpan.FromSeconds(15), TimeSpan.FromMilliseconds(500)).Until(condition =>
+              {
+                  try
+                  {
+                      var elementToBeDisplayed = driver.FindElement(locator);
+
```
Comment (AlexanderSavenok) [inline QAAutomationTraining/TUT.by/BasePage.cs]: Remove redundant empty line.

File: QAAutomationTraining/Test_With_Frames/FrameTests.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            driver.Quit();
+        }
+
+        [Test]
+        public void EnterTextWithDifferentFontTest()
+        {
+            Frame frame = new Frame(driver);
+            frame.ClearTextArea();
+
+            Assert.True(frame.TextAreaIsCleared());
```
Comment (AlexanderSavenok) [inline QAAutomationTraining/Test_With_Frames/FrameTests.cs]: Please, add information message for all assert methods.

File: QAAutomationTraining/Test_For_Alerts/AlertsTests.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        [Test]
+        public void JsPromptTest()
+        {
+            const string textForAlert = "text for alert";
+
+            AlertPage alertPage = new AlertPage(driver);
+            alertPage.ClickForJSPrompt(textForAlert);
+
+            Assert.True(alertPage.JsPromptIsClickedWithText(textForAlert));
```
Comment (AlexanderSavenok) [inline QAAutomationTraining/Test_For_Alerts/AlertsTests.cs]: Please, do not forget to add informative messages for assert methods in future.

File: QAAutomationTraining/TUT.by/BasePage.cs (hunk 1 above)
Comment (MarinaGotka) [reply QAAutomationTraining/TUT.by/BasePage.cs]: Shouldn't there be an empty string before return?

**Pull request merged**